src/main/java/Listener.java

```diff
@@ -18,8 +18,23 @@ private void conlin(MessageReceivedEvent event) {
             "> The limit of the function never depends on the value of the function at that point.\n\n- John Conlin"
         );
     }
-    private void german(MessageReceivedEvent event){
-        sendMessage(event, "Guten Morgen");
+    private void foreign(MessageReceivedEvent event){
+        int check  = (int)(Math.Random()*4+1);
+        swtich(check){
+            case 1:
+                sendMessage(event,"Guten Morgen!");
+                break;
+            case 2:
+                sendMessage(event,"eins ,zwei ,drei!");
+                break;
+            case 3:
+                sendMessage(event, "any preguntas?");
+                break;
+            case 4:
+                sendMessage(event,"Hasta La Pasta!");
```

Comment:
break;

Reply:
technically don't need it but nice to have for if you are adding more later on (future proofing)

Reply:
aight

Reply:
stop telling him what he doesn't need to put, hes gonna put another 4-1+1 thing in